src/web/app/src/components/BannerButtons.tsx

```diff
@@ -88,7 +87,7 @@ const BannerButtons = () => {
           disagreeButtonText="CANCEL"
         />
       )}
-      <Link href={`${webUrl}/docs/overview/`} passHref>
+      <Link href="/docs/overview" passHref>
```

Comment:
@Kevan-Y do you know why we did the `webUrl` here?  Does this change seem OK to you?

Reply:
Don't remember why we put that...
But this change seems to be okay.